cicecore/cicedynB/general/ice_forcing.F90

```diff
@@ -3741,7 +3724,10 @@ subroutine ocn_data_ncar_init
           enddo               ! month loop
         enddo               ! field loop
 
-        if (my_task == master_task) status = nf90_close(fid)
+        if (my_task == master_task) call ice_close_nc(fid)
```

Comment:
yes

Reply:
The master_task if is redundant.  I did not want to change the current implementation too much so I left it as is.  The redundancy is not a problem.  I agree this could be cleaned up.  But once you start down that path, there is a lot to do.  Just look at the whole ocn_data_ncar_init routine as one example.  My goal here was to change the nf90_close to a call to ice_close_nc because it was easy to do and then matches the ice_open_nc above.

Reply:
Ok, I understand.